package.json

```diff
@@ -10,18 +10,25 @@
   "scripts": {
     "build": "node_modules/.bin/webpack",
     "start": "node_modules/.bin/webpack-dev-server --host 0.0.0.0 --content-base public/ --hot",
-    "release": "node_modules/.bin/webpack -p"
+    "release": "node_modules/.bin/webpack",
+    "lint": "node_modules/.bin/eslint index.js"
   },
   "author": "Clappr team",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "shaka-player": "v2.0.9",
-    "clappr": "0.2.66"
+    "shaka-player": "2.0.8",
```

Comment:
`v2.0.9` doesn't exist in NPM, it doesn't seem to be a valid release actually? 

`2.0.8` is the latest release on the `2.0.x` minor.